xdsl/interactive/app.py

```diff
@@ -126,6 +151,16 @@ def on_mount(self) -> None:
 
         self.query_one("#input_container").border_title = "Input xDSL IR"
         self.query_one("#output_container").border_title = "Output xDSL IR"
+        self.query_one(
+            "#passes_selection_list"
+        ).border_title = "Choose a pass or multiplepasses to be applied."
+
+        # aids in the construction of the seleciton list containing all the passes
+        selections = [(value.name, value) for value in self.list_of_passes]
+        selections.sort()
```

Comment:
```suggestion
        selections = sorted((value.name, value) for value in self.list_of_passes)
```
If we're not providing a way for clients to specify the list of available passes for now, we can just reuse the function that generates them, without a class variable.